### PR TITLE
docs: add documentation on OpenCV reference

### PR DESCRIPTION
**Pull request by targos**
Plus reorder TestImagePath type alphabetically.

### CODE REVIEW

File: test/img/opencv/generate.py [hunk 1]
```diff
@@ -10,67 +16,65 @@
 M = np.float32([[scale, 0, 0], [0, scale, 0]])
 dst = cv.warpAffine(img, M, dsize=(cols * scale, rows * scale), flags=cv.INTER_LINEAR, borderMode=cv.BORDER_CONSTANT,
                     borderValue=0)
-cv.imwrite('testScale.png', dst)
+writeImg('testScale.png', dst)
 
-# Image resizing by 10.
+# Image resizing.
```
Comment (targos) [inline test/img/opencv/generate.py:21]: I plan to add more tests later, so it won't be only x10.

File: Development.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+To add a new test reference file:
+
+- Update [`generate.py`](./test/img/opencv/generate.py) with the OpenCV code that creates the file.
+- Run `generate.py` (see following paragraph).
+- Add the new filename to the [`TestImagePath`](./test/TestImagePath.ts) type (alphabetical order).
+
+To run the generation script, use the following steps:
+
+- Install Python 3.x
+- Run `python -m venv .venv` to create a virtual environment for the project
```
Comment (tpoisseau) [inline Development.md:21]: on mac with `brew install python`
`python` bin does not exists, we must use `python3`

Once the `venv` is active, `python` is working

**Review by tpoisseau — APPROVED** [review]
maybe a little adjustment in doc, but commands are working

File: Development.md (hunk 2 above)
Comment (targos) [inline Development.md:21]:
```suggestion
- Run `python3 -m venv .venv` to create a virtual environment for the project
```

**Pull request merged**